### PR TITLE
cal: Add use case for spanning date

### PR DESCRIPTION
**Pull request by nzachow**
For clarity I changed "-m" to "--monday".
Also, I've removed the "cal -y" for simplicity.
You could just user "cal {year}" to get the same information.

----

~~- [ ] The page (if new), does not already exist in the repo.~~

~~- [ ] The page (if new), has been added to the correct platform folder:  ~~
~~      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.~~

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by nzachow** [comment]
On Arch linux cal from util-linux 2.31 works as intended.
[Online documentation](http://man7.org/linux/man-pages/man1/cal.1.html) also suggests that these options should work.

What you think is the best approach here?

**Comment by agnivade** [comment]
Seems like ubuntu is taking cal from the freebsd variant https://www.unix.com/man-page/freebsd/1/cal/. Since we already have the proper page for cal under the osx folder. This can be more linux like.

File: pages/linux/cal.md [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
+- Display previous, current and next month:
 
-`cal -m`
+`cal --three`
 
-- Display a calendar for the current year:
+- Use monday as the first day of the week:
 
-`cal -y`
+`cal --monday`
```
Comment (agnivade) [inline pages/linux/cal.md]: Please wrap monday in tokens ({{}})

Comment (nzachow) [reply pages/linux/cal.md]: Sorry, but I'm not sure I understood what you meant.
I got the impression that you want this: `cal {{--monday}}`
Is this right ?

Comment (agnivade) [reply pages/linux/cal.md]: cal --{{monday}}. https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#token-syntax

Comment (nzachow) [reply pages/linux/cal.md]: This wouldn't cause confusion as someone could think that `{{monday}}` could be swapped for any other day ?

Comment (agnivade) [reply pages/linux/cal.md]: Oh I see that only monday or sunday is applicable. I initially thought any day can be used.

**Comment by agnivade** [comment]
ping @sbrl

**Comment by sbrl** [comment]
Thanks for the PR, @nzachow 😺

**Pull request merged**